internal/cli/daemon/daemon_test.go

```
@@ -0,0 +1,301 @@
+	Errors []struct {
+		Message string `json:"message"`
+	} `json:"errors"`
+}
+
+func postQuery(t *testing.T, url, doc string) gqlResponse {
+	t.Helper()
+	body, err := json.Marshal(map[string]string{"query": doc})
+	if err != nil {
+		t.Fatalf("marshal: %v", err)
+	}
+	resp, err := http.Post(url+"/graphql", "application/json", bytes.NewReader(body))
```

golangci-lint (2.12.1)

[error] net/http.Post must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)